### ISSUE TITLE
Training on Pose and bounding boxes and validating only on bounding boxes 

### ISSUE THREAD

**Issue by Faisal-Hajari** [ultralytics/ultralytics#17917]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I'm trying to train YOLOv11-pose. however, my training data is small and I want to train on pose and bounding boxes (pose will serve as an auxiliary task) and validation is only bounding boxes. I looked into it, but there isn't any docs about it or issues. is this supported ?

**Comment by Y-T-G** [comment]
You can refer to this

https://y-t-g.github.io/tutorials/yolov8-increase-accuracy/

**Comment by Faisal-Hajari** [comment]
really interesting. I might just add fake key-points to the validation set then and train. thanks

**Comment by glenn-jocher** [comment]
@Faisal-Hajari you're welcome! Adding fake keypoints for validation is a creative workaround. Just ensure consistency between your dataset structure and YOLO11's expected format to avoid issues during training and validation. Good luck!
